code/WorkInProgress/pomf/spacepods/spacepods.dm

```diff
@@ -326,24 +326,47 @@
 			if((A == src) || istype(A, /mob))
 				continue
 			return
-	move_outside(usr, T)
+	if(occupant == usr)
+		move_outside(usr, T)
+	else if(passengers.Find(usr))
+		move_passenger_outside(usr, T)
 
 /obj/spacepod/proc/move_outside(mob/living/user, turf/exit_loc = src.loc)
 	if(occupant)
 		inertia_dir = 0 // engage reverse thruster and power down pod
 		occupant.forceMove(exit_loc)
 		occupant = null
 		to_chat(usr, "<span class='notice'>You climb out of the pod.</span>")
+		if(passengers.len)
+			for(var/mob/living/L in passengers.len)
+				if(!occupant)
+
+					var/answer = alert(L,"The pilot has evacuated \the [src]. Do you wish to take over?",,"Yes","No")
```

Comment:
How abouts do I add verbs only under certain conditions? I tried occupant.verbs += (verb), but that only seems to work for verbs defined under the subtype of the occupant, and not the spacepod.

Verbs seem like quite a mess, hence I want to UI it up a bit